docs/io.cozy.files.md

```diff
@@ -284,3 +284,9 @@ fields), and a relationship to the versioned file.
   }
 }
 ```
+
+## `io.cozy.files.settings`
+
+This doctype is used to store settings for files. It is currenly only used for the qualification migration service.
```

Comment:
I think it is weird to call that `settings`: it is not something that the user can set, more an internal detail.